### ISSUE TITLE
Is this project still in progress ?

### ISSUE THREAD

**Issue by maisnamraju** [dangular/storeroom83#1]
Just curious as the last commit was a year ago

**Comment by dangular** [comment]
No.  I was pulled into a big project for a new client that hasn't allowed any time to focus on this over the past year.

**Comment by ferngarc** [comment]
shucks. Come back Dan! I'm currently learning how to develop using MEAN and this is a great stepping stone for me.

**Comment by lcschiltz** [comment]
Hello Dangular,
My former company is looking for someone to develop a product inventory management for them. Would you be available for a freelancing mission ?
